packages/bitvec/bitvec.2.4.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "bitvec"
+version: "2.4.0"
+maintainer: "Ivan Gotovchits <ivg@ieee.org>"
+authors: "BAP Team"
+homepage: "https://github.com/BinaryAnalysisPlatform/bap/"
+bug-reports: "https://github.com/BinaryAnalysisPlatform/bap/issues"
+dev-repo: "git://github.com/BinaryAnalysisPlatform/bap/"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%" "--enable-bitvec"]
+  [make]
+]
+install: [[make "install"]]
+remove: [["ocamlfind" "remove" "bitvec"] ]
+depends: [
+  "ocaml" {>= "4.08.0" }
+  "oasis" {build & >= "0.4.7"}
+  "zarith"
```

Comment:
You need a lower bound here
```suggestion
  "zarith" {>= "1.4"}
```

Error:
```
#=== ERROR while compiling bitvec.2.4.0 =======================================#
# context              2.1.1 | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.08/.opam-switch/build/bitvec.2.4.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build make
# exit-code            2
# env-file             ~/.opam/log/bitvec-41-e01835.env
# output-file          ~/.opam/log/bitvec-41-e01835.out
### output ###
# ./setup.exe -quiet -build 
# ocamlfind ocamlopt -package unix -package ocamlbuild -linkpkg -package findlib myocamlbuild.ml /home/opam/.opam/4.08/lib/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
# /home/opam/.opam/4.08/bin/ocamlfind ocamldep -modules tools/postinstall.ml > tools/postinstall.ml.depends
# /home/opam/.opam/4.08/bin/ocamlfind ocamlc -c -g -annot -bin-annot -w +a-4-6-7-9-27-29-32..42-44-45-48-50-60 -I tools -o tools/postinstall.cmo tools/postinstall.ml
# /home/opam/.opam/4.08/bin/ocamlfind ocamlopt -c -g -annot -bin-annot -w +a-4-6-7-9-27-29-32..42-44-45-48-50-60 -I tools -o tools/postinstall.cmx tools/postinstall.ml
# /home/opam/.opam/4.08/bin/ocamlfind ocamlopt -linkpkg -g -linkpkg -I tools tools/postinstall.cmx -o tools/postinstall.native
# /home/opam/.opam/4.08/bin/ocamlfind ocamldep -package zarith -modules lib/bitvec/bitvec.mli > lib/bitvec/bitvec.mli.depends
# /home/opam/.opam/4.08/bin/ocamlfind ocamlc -c -g -annot -bin-annot -w +a-4-6-7-9-27-29-32..42-44-45-48-50-60 -package zarith -I lib/bitvec -o lib/bitvec/bitvec.cmi lib/bitvec/bitvec.mli
# /home/opam/.opam/4.08/bin/ocamlfind ocamldep -package zarith -modules lib/bitvec/bitvec.ml > lib/bitvec/bitvec.ml.depends
# /home/opam/.opam/4.08/bin/ocamlfind ocamlc -c -g -annot -bin-annot -w +a-4-6-7-9-27-29-32..42-44-45-48-50-60 -package zarith -I lib/bitvec -o lib/bitvec/bitvec.cmo lib/bitvec/bitvec.ml
# + /home/opam/.opam/4.08/bin/ocamlfind ocamlc -c -g -annot -bin-annot -w +a-4-6-7-9-27-29-32..42-44-45-48-50-60 -package zarith -I lib/bitvec -o lib/bitvec/bitvec.cmo lib/bitvec/bitvec.ml
# File "lib/bitvec/bitvec.ml", line 112, characters 17-26:
# 112 |       let y = if Z.testbit x v
#                        ^^^^^^^^^
# Error: Unbound value Z.testbit
# Command exited with code 2.
# make: *** [Makefile:8: build] Error 1
```

Reply:
awesome, thanks! I always wanted to know what is the lower bound in zarith :)